python/MooseDocs/extensions/content.py

```diff
@@ -162,6 +163,25 @@ def createToken(self, parent, info, page):
                                levels=levels,
                                columns=int(self.settings['columns']))
 
+class ContentOutlineCommand(command.CommandComponent):
+    COMMAND = 'content'
+    SUBCOMMAND = 'outline'
+
+    @staticmethod
+    def defaultSettings():
+        settings = command.CommandComponent.defaultSettings()
+        settings['location'] = (None, "The markdown content directory to build outline.")
+        settings['level'] = (2, 'Heading level(s) to display.')
+        return settings
```

Comment:
I think it would be nice to have the `'hide'` setting here, no?

Reply:
It wouldn't hurt.